python/fledge/services/core/api/plugins/remove.py

```diff
@@ -19,6 +19,7 @@
 from fledge.common.configuration_manager import ConfigurationManager
 from fledge.common.common import _FLEDGE_ROOT
 from fledge.common.audit_logger import AuditLogger
+from fledge.services.core.server import Server
```

Comment:
Move this import in function scope, As we have seen in past import Server statement giving us
circular import issue & to disallows many unit tests from running.